apps/docs-next/src/content/learn/examples/geometry/terrain.mdx

```diff
@@ -4,4 +4,61 @@ title: Terrain with 3D noise
 showInSidebar: false
 ---
 
+### Why
+
+The key to creating "random" yet smooth terrain is using "noise".
+
+To create a terrain map, we want to be able to input the x and y coordinate , and return a height value.
+Therefore we want to utilise 2D noise, and this example uses `createNoise2D` from npm package 'simplex-noise'.
```

Comment:
"and this example uses the function `createNoise2D` from the package `simplex-noise`."